themes/default/layouts/challenge/list.html

```diff
@@ -0,0 +1,76 @@
+            t-shirt.
+        </p>
+    </section>
+
+    <section id="challenges" class="w-full relative overflow-visible">
+        <!-- Only do the 2x2 layout at tablet width (md) and up. -->
+        <ul class="md:flex flex-wrap list-none">
+            <li class="md:w-1/2">
+                <div>
+                    <div class="w-full p-4 my-2">
+                        <div class="card text-center py-16 px-8 relative bg-white">
+                            <h4>Startup in a Box</h4>
+                            <p class="my-8">Build and deploy your very own landing site to the cloud, complete with Cloudfront and Checkly!</p>
+                            <div class="flex flex-col lg:flex-row justify-center items-center">
+                                <div class="my-2 mx-2 w-full lg:w-auto">
+                                    <a href="{{ relref . "/startup-in-a-box" }}" class="w-full block btn-secondary">Start</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </li>
+            <li class="md:w-1/2">
+                <div>
+                    <div class="w-full p-4 my-2">
+                        <div class="card text-center py-16 px-8 relative bg-white">
+                            <h4>Templating for Fun and Profit</h4>
+                            <p class="my-8">Launching in September!</p>
+                            <div class="flex flex-col lg:flex-row justify-center items-center">
+                                <div class="my-2 mx-2 w-full lg:w-auto">
+                                    <a href="" class="w-full block btn-secondary">Coming Soon</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </li>
+            <li class="md:w-1/2">
+                <div>
+                    <div class="w-full p-4 my-2">
+                        <div class="card text-center py-16 px-8 relative bg-white">
+                            <h4>Overengineering Anything</h4>
+                            <p class="my-8">Launching in October!</p>
+                            <div class="flex flex-col lg:flex-row justify-center items-center">
+                                <div class="my-2 mx-2 w-full lg:w-auto">
+                                    <a href="" class="w-full block btn-secondary">Coming Soon</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </li>
+            <li class="md:w-1/2">
+                <div>
+                    <div class="w-full p-4 my-2">
+                        <div class="card text-center py-16 px-8 relative bg-white">
+                            <h4>TBD</h4>
```

Comment:
I would just make something up instead of leaving it TBD. Always can change it later. 

"Scaling on AWS" or something that is pretty vague